scheduler/src/cook/config_incremental.clj

```diff
@@ -99,10 +99,18 @@
   (select-config-from-values uuid (read-config config-key)))
 
 (defn resolve-incremental-config
-  "Resolve an incremental config to the appropriate value."
-  [^UUID uuid incremental-config]
-  (cond
-    (coll? incremental-config)
-    (select-config-from-values uuid incremental-config)
-    (keyword? incremental-config)
-    (select-config-from-key uuid incremental-config)))
\ No newline at end of file
+  "Resolve an incremental config to the appropriate value.
+  With an overload that takes a fallback config to use in case the incremental config cannot be resolved."
```

Comment:
Should there be a warning if it was unable to be resolved? (If this is done elsewhere, probably worth a comment?)